docs/src/release_notes/v1.16.md

```diff
@@ -6,6 +6,55 @@ For a complete list of changes, please refer to the
 [commits](https://github.com/cloudnative-pg/cloudnative-pg/commits/release-1.16)
 on the release branch in GitHub.
 
+## Version 1.16.5
+
+**Release date:** Dec 21, 2022
+
+!!! Warning
+    Version 1.16 has reached End-of-Life (EOL). Version 1.16.5 is the last
+    that will be released for the 1.16 minor version.
+
+Important announcements:
+
+- Recognizing Armando Ruocco (@armru) as a new CloudNativePG maintainer for his
+  consistent and impactful contributions (#1167)
+- FINAL patch release for 1.16: 1.16.5. Release 1.16 reaches end of life.
+
+Enhancements:
+
+- Add rpm/deb package for kubectl-cnpg plugin (#1008)
+- Update default PostgreSQL version for new cluster definitions to 15.1 (#908)
+- Remove ARMv7 support (#1092)
```

Comment:
I do not see this fitting the Enhancements section. I would move it in the announcements

Reply:
right you are, will move